### DISCUSSION TITLE
How to use a socks proxy with VSCodium?

### DISCUSSION THREAD

**Discussion by gerroon** [VSCodium/vscodium#1755] [Q&A]
Is it possible to use socks5 (or 34 at least) proxy with VSCodium?

thanks

**Comment by daiyam** [comment]
If you want to connect VSCodium to a remote server, I recommend the extension [Open Remote - SSH
Preview](https://open-vsx.org/extension/jeanp413/open-remote-ssh).
Then you will need to wrap your ssh connection around Socks5 (maybe with the `ALL_PROXY` env variable).

**Reply by gerroon** [reply]
Thanks but I need to use  a socks server to connect to the extension market.
